### PR TITLE
add Enchantments to mesh

### PR DESCRIPTION
**Pull request by remplerus**
make it draft as i'm not 100% sure if this one fits well

### CODE REVIEW

**Comment by remplerus** [comment]
also fixes #136

File: src/main/java/novamachina/exnihilosequentia/common/tileentity/SieveTile.java [hunk 1]
```diff
@@ -168,14 +172,14 @@ public void activateSieve(PlayerEntity player, boolean isWaterlogged) {
...
-            progress += 1;
+            progress += 1 * (1+efficiency/5);
 
             if (progress >= Config.getMaxSieveClicks()) {
                 logger.debug("Sieve progress complete");
                 List<SieveRecipe> drops = ExNihiloRegistries.SIEVE_REGISTRY
                     .getDrops(((BlockItem) blockStack.getItem()).getBlock(), meshType, isWaterlogged);
                 drops.forEach((entry -> entry.getRolls().forEach(meshWithChance -> {
-                    if (random.nextFloat() <= meshWithChance.getChance()) {
+                    if (random.nextFloat() <= meshWithChance.getChance() * (1F + (fortune/3))) {
```
Comment (remplerus) [inline src/main/java/novamachina/exnihilosequentia/common/tileentity/SieveTile.java:182]: this one is changable

File: src/main/java/novamachina/exnihilosequentia/common/tileentity/SieveTile.java [hunk 2]
```diff
@@ -168,14 +172,14 @@ public void activateSieve(PlayerEntity player, boolean isWaterlogged) {
         }
 
         if (isReadyToSieve()) {
-            progress += 1;
+            progress += 1 * (1+efficiency/5);
```
Comment (remplerus) [inline src/main/java/novamachina/exnihilosequentia/common/tileentity/SieveTile.java:175]: this one is changable

File: build.gradle [hunk 3]
```diff
@@ -63,9 +62,6 @@ dependencies {
     compileOnly fg.deobf("mezz.jei:jei-${minecraft_version}:${jei_version}:api")
     runtimeOnly fg.deobf("mezz.jei:jei-${minecraft_version}:${jei_version}")
 
-    compileOnly fg.deobf("mcp.mobius.waila:Hwyla:${hwyla_version}_${minecraft_release}.2:api")
-    //runtimeOnly fg.deobf("mcp.mobius.waila:Hwyla:${hwyla_version}_${minecraft_release}.2")
```
Comment (NovaMachina) [inline build.gradle:67]: 1.16 still uses HWYLA, so this should not be removed.

**Review by NovaMachina — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 3 above)
Comment (remplerus) [reply build.gradle:67]: this isn't removed, i just chaged it to jade's api which is same as hwyla for 1.16

Comment (NovaMachina) [reply build.gradle:67]: So the Jade API is absolutely identical to HWYLA? Have you checked that both Jade and HWYLA work still?

Comment (remplerus) [reply build.gradle:67]: yes

**Review by NovaMachina — APPROVED** [review]

**Pull request merged**